### PR TITLE
DevOps Reading List

### PR DESCRIPTION
**Pull request by wordlee**

### CODE REVIEW

**Comment by Alex-Yates** [comment]
Great edits. Thanks as always.

Just one comment above about the genesis sentence, but that's more to do with my writing than your edit.

Thanks again. :-)

File: blog/2020-11/devops-reading-list/index.md [hunk 1]
```diff
@@ -172,11 +166,11 @@ The Poppendiecks take seven of the core lean principles and talk in practical te
 1.	Build Integrity In
 1.	See the Whole
 
-Along the way they cover 20 tools for applying the lean principles into practice, such as Value-Stream Mapping, working in iterations and refactoring.
+Along the way, they cover 20 tools for applying the lean principles into practice, such as Value-Stream Mapping, working in iterations, and refactoring.
 
-This book stands up as one of (if not the) best, articulation of what Agile actually is, how it builds on stable Lean foundations, and why specific practices result in better software delivery outcomes. As an “Agile” book, it’s not technically a “DevOps” book (it was written six years before Patrick DeBois accidentally gave us *that* buzzword) but in many ways, this is DevOps’ genesis.
+This book stands up as one of the (if not the) best articulation of what Agile actually is, how it builds on stable Lean foundations, and why specific practices result in better software delivery outcomes. As an “Agile” book, it’s not technically a “DevOps” book (it was written six years before Patrick DeBois accidentally gave us *that* buzzword), but in many ways, this is the genesis of DevOps.
```
Comment (wordlee) [inline blog/2020-11/devops-reading-list/index.md:171]:
```suggestion
This book stands up as one of the (if not the) best articulation of what Agile actually is, how it builds on stable Lean foundations, and why specific practices result in better software delivery outcomes. As an “Agile” book, it’s not technically a “DevOps” book (it was written six years before Patrick DeBois accidentally gave us *that* buzzword), but this book was an important chapter in the genesis of DevOps..
```

**Comment by wordlee** [comment]
@Alex-Yates I'm merging this to master where I'll work on the image alignment.

**Comment by wordlee** [comment]
I cheated to dodge the merge conflict. The post is now on master.

**Pull request closed without merging**